### PR TITLE
Implement new otp backend

### LINKED ISSUE #49: unable to login
I cannot manage to login : 

.....BeFake/BeFake/BeFake.py", line 99, in send_otp
    self.otp_session = res["sessionInfo"]
                       ~~~^^^^^^^^^^^^^^^
KeyError: 'sessionInfo'

### PR DESCRIPTION
**Pull request by VxlerieUwU**
This fixes #49 for me and allows not dealing with iOS receipts and other stuff. I've put the old google api as a fallback in case if the new vonage backend goes offline.

### CODE REVIEW

**Comment by VxlerieUwU** [comment]
This also needs a new deviceId parameter that is a random lowercase alphanumeric string that I generate inside of the click argument option, to allow users to put their own. It's only used for vonage otp auth here.

**Comment by notmarek** [comment]
You are doing awesome work would you like write access to the repo?

**Comment by VxlerieUwU** [comment]
> You are doing awesome work would you like write access to the repo?

Why not, I still need to rewrite all cloudfunctions.net requests. Thank you very much  😃

**Comment by notmarek** [comment]
> > You are doing awesome work would you like write access to the repo?
> 
> 
> 
> Why not, I still need to rewrite all cloudfunctions.net requests. Thank you very much  😃

Invite sent, make sure to plaster your name all over here as most of it is your code by now

File: BeFake/__main__.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import json
 import os
+
 from functools import wraps
 
+import string
+import random
+
+from .BeFake import BeFake
```
Comment (NicoWeio) [inline BeFake/__main__.py:9]: `BeFake` is imported twice now.

Comment (VxlerieUwU) [reply BeFake/__main__.py:9]: OMG I didn't see that until now 😂

Comment (NicoWeio) [reply BeFake/__main__.py:9]: I mean, it probably doesn't break anything. 😄
Note to future readers: Fixed in 9e8ff28e95ae71fd2afd6cb1dd4c4791ed109c9e.

**Pull request merged**